modules/ROOT/pages/provisioning-ibmcloud.adoc

```diff
@@ -71,7 +72,9 @@ ibmcloud cos upload --bucket=$BUCKET --key=$FILE --file=$FILE
 ----
 IMAGE=${FILE:0:-6}     # pull off .qcow2
 IMAGE=${IMAGE//[._]/-} # replace . and _ with -
-ibmcloud is image-create $IMAGE --file "cos://${REGION}/${BUCKET}/${FILE}" --os-name fedora-coreos-stable-amd64
+[ $ARCH == 'x86_64' ] && OSNAME='fedora-coreos-stable-amd64'
+[ $ARCH == 's390x' ] && OSNAME='red-8-s390x-byol'
```

Comment:
Feels like this deserves some explanation. E.g. something like "Fedora CoreOS is not yet a valid s390x OS name in IBM cloud, so we use RHEL8's"? (I presume that's what's happening here.)

Reply:
added some context in a `NOTE:`.